### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by fejoa**
I've felt for some time now that we could have a guideline for contributing. Here's my suggestion for a contribution guideline, based on my observation of how how people have been contributing over the years. It's not exhaustive by any means, please feel free to make any changes based on your own observations and collaboration experience. As usual, there is no time frame for completion.

One feature I would like to see is if a PR is approved, then anyone including the author can merge that PR. Not sure if this is useful, but it feels like this builds consensus and is more laid back. I don't know how anyone else feels about merging ones own work, but maybe there are circumstances which merit this?

I think we can mention the useful feature of GitHub's draft PR to distinguish between work in progress and stuff that is ready to be reviewed/merged.

There's also a things to do, a job list, and a forbidden jobs. Upsetting the balance is probably the worst, after coding bugs. I think there are probably enough materials and weapons at this stage in development.

### CODE REVIEW

**Comment by iology** [comment]
I only have issue about squashing commits when merging:

* Is it okay to rebase after approval and does this require another careful review.
* Only squash long and messy commit history?

**Comment by AquariusPower** [comment]
> if a PR is approved, then anyone including the author can merge that PR. 

anyone?

just some thoughts...
It should be limited to the last commit til the approval (that should be an action/trigger/checkbox detected by github allowing one merge only, and not a mere comment even from a recognized member), so new commits would require re-approval (only members can do that), but I dont really know how much can be configured in that way...

**Comment by andrewtweber** [comment]
Thanks for putting this together @fejoa 

> anyone?

@AquariusPower I think only people who are assigned to the [Developers team](https://github.com/orgs/Attnam/people) on the Attnam organization have permission to merge. Maybe the documentation should be more clear on that.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ - Commenting the code, document the danger system, weapon balance and combat system.
+
+__Forbidden jobs__
+
+ - Intentionally creating bugs.
+ - Upsetting game balance.
+ - Adding new materials.
+ - Adding/modding weapons without knowing how the weapon system is balanced.
+ - Adding content that breaches the [Code of Conduct](https://github.com/Attnam/ivan/blob/master/CODE_OF_CONDUCT.md).
+ - Telepathically communicating with Elpuri
```
Comment (red-kangaroo) [inline CONTRIBUTING.md:27]: Missing fullstop. :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ - Coding new subsystems, for example bodypart mutation subsystem, new dangers for player characters.
+ - Improving continuous integration and both cloud- and ground-based compiling systems for all supported platforms.
+ - Scripting new dungeons and new places for players to go.
+ - Commenting the code, document the danger system, weapon balance and combat system.
+
+__Forbidden jobs__
+
+ - Intentionally creating bugs.
+ - Upsetting game balance.
+ - Adding new materials.
```
Comment (red-kangaroo) [inline CONTRIBUTING.md:24]: Well, this should have a footnote: Except when necessary or aesthetically pleasing (like new colours of ground for Urok :P ).

File: CONTRIBUTING.md (hunk 1 above)
Comment (AquariusPower) [reply CONTRIBUTING.md:27]: Only a true hero can put a fullstop to Elpuri's threat! in a sense.. :>

**Comment by fejoa** [comment]
@red-kangaroo haha, yes thanks for pointing out the missing full stop. The new materials embargo is probably outmoded and I think we can dispense with it. (Yes, you got me, I am guilty of introducing asthetic materials as well :E )
I'll make some more changes when I next get to a terminal, hopefully by Tuesday.

**Comment by fejoa** [comment]
PR #605 should do the trick

**Pull request merged**